lib/tools.js

```diff
@@ -220,7 +220,9 @@ function prettifierMetaWrapper (pretty, dest) {
 
       var time = this.lastTime
 
-      if (time.match(/^\d+/)) {
+      if (!time) {
```

Comment:
It would be a good idea at this point to check that time actually has `match` in its string prototype for type restrictions before attempting to invoke it. This would reduce the nature of of "breaking" errors and malice code in the stream overflowing.